File: TUF/String/RemoveAllSpaces.java
import java.util.Scanner;

public class RemoveAllSpaces {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.print("Enter the string: ");
        String str = input.nextLine();
        input.close();
        System.out.println("Before remove all spaces the string is: " + str);
        System.out.println("After remove all spaces the string is: " + removeSpaces(str));
    }

    private static String removeSpaces(String str) {
        StringBuilder newString = new StringBuilder();
        for (int i = 0; i < str.length(); i++) {
            char currChar = str.charAt(i);
            if (currChar == ' ') {
                continue;
            }
            newString.append(currChar);
        }
        return newString.toString();
    }
}

File: TUF/String/RemoveAllSpaces.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
function removeSpaces(str) {
    var newStr = "";
    for (var i = 0; i < str.length; i++) {
        var currChar = str.charAt(i);
        if (currChar == ' ') {
            continue;
        }
        newStr += currChar;
    }
    return newStr;
}
var str = "Take you forward";
console.log("Before remove all spaces the string is: ".concat(str));
console.log("After remove all spaces the string is: ".concat(removeSpaces(str)));
